archetypes/oci-mp/src/main/resources/server/src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -0,0 +1,19 @@
+# Application properties. This is the default greeting
+app.greeting=Hello
+
+# Microprofile server properties
+server.host=0.0.0.0
+server.port=8080
+
+# Turn on support for REST.request SimpleTimers for all JAX-RS endpoints
+metrics.rest-request.enabled=true
+
+# OCI Monitoring configuration
+oci.monitoring.compartmentId=<your monitoring compartment id>
```

Comment:
can we file a task to add these to the quickstart interview please?

Reply:
We don't want to force end user to have their infra setup done before generating app. They can always do CRUD for config properties as required which is documented in README.